src/dndbeyond/content-scripts/encounter.js

```diff
@@ -25,11 +25,12 @@ function documentModified(mutations, observer) {
     character.parseStatBlock(monster);
 }
 
+let lastCombatUpdate = null;
```

Comment:
I'd use var in this case. As it's a global variable. I consider 'let' to be for small scope temp variables. I may be wrong in how people do it, but that's the convention I've been using.

Reply:
I'm personally against using deprecated keywords as a rule... but if you insist further I can do that (but it'll hurt).

Reply:
My understanding is that `var` isn't deprecated, but rather it's used for something different. Specifically, `let` is for the current block scope, same as `const` but `var` is for the function's scope or for global variables. People say to use `let` in favor of `var` because it makes it less error prone, as you may get confused about your variables and the variable may be in the wrong scope, etc... totally understandable and good avice. In this case however, we are declaring an actual global variable, which is why I've taken to declaring them with `var` to make it clear that **hey look, this is a global variable right here**.